packages/opentelemetry-context-async-hooks/package.json

```diff
@@ -43,7 +43,7 @@
     "access": "public"
   },
   "devDependencies": {
-    "@opentelemetry/api": "~1.1.0",
+    "@opentelemetry/api": "^1.0.0",
```

Comment:
Is it intended to allow a wider range in dev dependency as in peer dependency? Couldn't this result in a broken build once a new API is released?

Reply:
Yes I'll make them match